<?xml version="1.0"?>
<robot name="Tarobaban">

  <link name="base_link">
    <visual>
      <geometry>
        <box size="3 3 .01"/>
      </geometry>
       <origin rpy="0 0 0" xyz="0 0 -0.01"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
  </link>

<joint name="base_link_to_base_body" type="revolute">
    <parent link="base_link"/>
    <child link="base_body"/>
    <axis xyz="0 0 1"/>
    <origin xyz="0 0 0.01"/>
    <limit effort ="1000.0" lower="-2.0" upper="2.0" velocity="0.5"/>
  </joint>


  <link name="base_body">
    <visual>
      <geometry>
        <cylinder length="0.350" radius="0.08"/>
      </geometry>
       <origin rpy="0 0 0" xyz="0 0 0.175"/>
      <material name="blue">
        <color rgba="0 0 1 1"/>
      </material>
    </visual>
  </link>

  <link name="lower_arm">
    <visual>
      <geometry>
        <box size=".1 .05 .350"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0.175"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
  </link>

  <joint name="base_to_lower_arm" type="revolute">
    <parent link="base_body"/>
    <child link="lower_arm"/>
    <axis xyz="1 0 0"/>
    <origin xyz="0 0 0.350"/>
    <limit effort ="1000.0" lower="-2.0" upper="2.0" velocity="0.5"/>
  </joint>



  <link name="upper_arm">
    <visual>
      <geometry>
        <box size="0.479 .05 .05"/>
      </geometry>
      <origin rpy="0 1.57075 0" xyz="0 0 0.2395"/>
      <material name="red">
        <color rgba="1 0 0 1"/>
      </material>
    </visual>
  </link>

  <joint name="lower_arm_to_upper_arm" type="revolute">
    <parent link="lower_arm"/>
    <child link="upper_arm"/>
    <axis xyz="1 0 0"/>
    <origin xyz="0 0 0.350"/>
    <limit effort ="1000.0" lower="-2.0" upper="2.0" velocity="0.5"/>
  </joint>



</robot>